account_invoice_export/__manifest__.py

```diff
@@ -3,15 +3,16 @@
 
 {
     "name": "Account Invoice Export",
-    "version": "13.0.1.0.0",
+    "version": "14.0.1.0.0",
     "category": "Invoicing Management",
     "license": "AGPL-3",
     "summary": "",
     "author": "Camptocamp SA, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/edi",
-    "depends": ["account_invoice_transmit_method", "queue_job"],
+    "depends": ["sale", "account_invoice_transmit_method", "queue_job"],
```

Comment:
Is the `sale` module addition to the dependencies needed ?

Reply:
ah it was a mistake, fixed, thanks